packages/map-popup/src/MapPopup.story.tsx

```diff
@@ -64,13 +67,33 @@ const FLOATING_CAR = {
   y: 52.52
 };
 
+const getEntityPrefixExample = (entity: Stop | Station) => {
+  const DemoIcon = styled.span`
+    background-color: blue;
+    border-radius: 50px;
+    color: white;
+    margin-right: 0.5ch;
+    padding: 2px;
+  `;
+
+  return <DemoIcon>{entity.name?.charAt(0)}</DemoIcon>;
+};
+
 export const StopEntity = (): JSX.Element => (
   <MapPopupContents
     entity={STOP}
     setLocation={action("setLocation")}
     setViewedStop={action("setViewedStop")}
   />
 );
+export const StopEntitywithEntityPrefix = (): JSX.Element => (
+  <MapPopupContents
+    entity={STOP}
+    getEntityPrefix={() => getEntityPrefixExample(STOP)}
```

Comment:
I think, per `packages/map-popup/src/index.tsx`, this attribute can be simplified to `getEntityPrefix={getEntityPrefixExamnple}`.